cnf-tests/testsuites/pkg/features/features.go

```diff
@@ -206,3 +206,29 @@ func (p *BondcniFixture) Setup() error {
 func (p *BondcniFixture) Cleanup() error {
 	return namespaces.Delete(namespaces.BondTestNamespace, testclient.Client)
 }
+
+type MultiNetworkPolicyFixture struct {
```

Comment:
should we also call sriov clean here?

Reply:
this comment seems to be a duplicated of [this](https://github.com/openshift-kni/cnf-features-deploy/pull/1113#discussion_r953913952)